modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/DefinitionOfTermsContainer/DefinitionOfTermsContainer.tsx

```diff
@@ -48,6 +48,17 @@ export default function DefinitionOfTermsContainer({
 
 	return (
 		<Card title={Liferay.Language.get('definition-of-terms')}>
+			{hasLocalizedField(objectDefinitions, selectedEntityId) && (
```

Comment:
i think that calling this function directly may not be so good, because it will execute every time the component re-renders; if the object has a lot of fields, this could lead to performance issues (even tho this would be hard in this case, but is better to prevent this kind of scenario if possible)

what do you think to create a useMemo for that instead of the function? on the array dependency it would have the objectDefinitions and selectedEntityId, with the value would only changing when the prop changes